src/scripts/crank.ts

```diff
@@ -162,6 +165,9 @@ async function run() {
   // noinspection InfiniteLoopJS
   while (true) {
     try {
+      let crankInstructionsQueue: TransactionInstruction[] = [];
+      let instructionBumpMap = new Map();
```

Comment:
you can use a simple boolean flag, or pull up the same one "shouldBumpFee" you use below.